src/main/java/com/github/steveice10/mc/protocol/data/game/chunk/Column.java

```diff
@@ -13,17 +13,13 @@
     private final @NonNull Chunk[] chunks;
     private final @NonNull CompoundTag[] tileEntities;
     private final @NonNull CompoundTag heightMaps;
-    private final int[] biomeData;
+    private final @NonNull int[] biomeData;
 
     public Column(int x, int z, @NonNull Chunk[] chunks, @NonNull CompoundTag[] tileEntities, @NonNull CompoundTag heightMaps) {
-        this(x, z, chunks, tileEntities, heightMaps, null);
+        this(x, z, chunks, tileEntities, heightMaps, new int[0]);
```

Comment:
According to the wiki this should be size 1024, not 0. In fact this constructor should probably be at least deprecated since biome data is now required.